storage/src/mmr/verification.rs

```diff
@@ -429,120 +436,93 @@ mod tests {
         // test range proofs over all possible ranges of at least 2 elements
         let root_hash = mmr.root();
         let mut hasher = Sha256::default();
-        for i in 0..elements.len() {
-            for j in i + 1..elements.len() {
-                let start_pos = element_positions[i];
-                let end_pos = element_positions[j];
-                let range_proof = mmr.range_proof(start_pos, end_pos).unwrap();
-                assert!(
-                    range_proof.verify_range_inclusion(
-                        &mut hasher,
-                        &elements[i..j + 1],
-                        start_pos,
-                        end_pos,
-                        &root_hash,
-                    ),
-                    "valid range proof should verify successfully {}:{}",
-                    i,
-                    j
-                );
+
+        let (executor, _, _) = Executor::default();
```

Comment:
in other tests, I just do this at the start (even if it surrounds sync work).

Reply:
done